src/components/freelancerJobs/freelancerPage.tsx

```diff
@@ -50,7 +50,21 @@ const FreelancerPage: FC = () => {
 	};
 
 	useEffect(() => {
-		if (!isLoading) {
+		if (!isLoading && userInfo) {
+			const category = {
+				value: userInfo.profile.category.name,
+				label: userInfo.profile.category.name,
+			};
+			const skills = useMemo(
+				() =>
+					skillsMock.map(skill => ({
+						...skill,
+						value: userInfo.profile.skills.some(
+							(jobSkill: { name: string }) => jobSkill.name === skill.name,
+						),
+					})),
+				[skillsMock],
```

Comment:
does it possible to make it more simple? we have 2 loop here + caching, I don't think this complexity is needed